xapian-bindings/configure.ac

```diff
@@ -94,6 +94,24 @@ XO_LIB_XAPIAN([], [],
 			  [^\([0-9]*\.[0-9]*[13579]\)\..*$], [-\1]))
 XO_REQUIRE([1.4.0])
 
+AC_ARG_ENABLE([letor],
+  [AS_HELP_STRING([--enable-letor], [])],
+  [case ${enableval} in
+    yes) ;;
+    no)
+      AC_MSG_WARN([xapian-letor is unavailable, hence letor-bindings won't be built])
+      ;;
+    *) AC_MSG_ERROR([bad value ${enableval} for --enable-letor]) ;;
+  esac],
+  [enable_letor=no])
+
+AM_CONDITIONAL([LETOR_FOUND], [test yes = "$enable_letor"])
+
+if test yes = "$enable_letor"; then
+    dnl Check for xapian-letor.
+    XO_LIB_XAPIANLETOR([], [], [xapianletor-config])
```

Comment:
The existing shell code in this file uses a 2 space indent, but here (and below) you're using 4 spaces.

Reply:
fixed in a fixup commit.